spec/doorkeeper-jwt/doorkeeper-jwt_spec.rb

```diff
@@ -139,5 +139,26 @@
       expect(decoded_token[1]["typ"]).to eq "JWT"
       expect(decoded_token[1]["alg"]).to eq "RS512"
     end
+
+    it "creates a signed JWT token encrypted with an application key from a string" do
+      secret_key = OpenSSL::PKey::RSA.new(1024)
+      Doorkeeper::JWT.configure do
+        use_application_secret true
+        token_payload do |opts|
```

Comment:
Unused block argument - `opts`. You can omit the argument if you don't care about it.